packages/react-scripts/config/paths.js

```diff
@@ -53,7 +53,6 @@ module.exports = {
   appIndexJs: resolveApp('src/index.js'),
   appPackageJson: resolveApp('package.json'),
   appSrc: resolveApp('src'),
-  testSrc: resolveApp('tests'),
```

Comment:
Why remove it?

Reply:
It's not used anywhere anymore, will be easier to get updates from upstream if we don't leave things that aren't needed anymore.